src/components/shared/progressbar.tsx

```diff
@@ -0,0 +1,37 @@
+import './style.css';
+
+function Bar(): JSX.Element {
+  return (
+    <div>
+      <div id="progress">
+        <div id="progress-bar"></div>
+        <ul id="progress-num">
+          <li className="step active">1</li>
```

Comment:
This looks great, but we don't want it to always have the first step active, and also we don't want it to only have 4 steps. Pass the number of steps and the current step in as props to the component and create your < li>'s accordingly.